lib/components/viewers/pattern-row.tsx

```diff
@@ -101,6 +103,11 @@ const PatternRow = ({
             />
           </span>
           <span style={{ wordBreak: 'break-word' }} title={pattern.headsign}>
+            {alwaysShowLongName && !!pattern.route.longName && (
```

Comment:
I like the idea of avoiding the crash and having layout issues. Maybe insert a comment here, warning about the potential layout issues to compliment the comment in the config?

Reply:
Good idea